### PR TITLE
Disable verbose output in sketch compilation CI logs

### PR DESCRIPTION
**Pull request by per1234**
The `arduino/compile-sketches` action was previously configured for verbose output.. This option is primarily intended to be used for troubleshooting and doesn't contain any information that is useful for general usage.

Due to the extensive coverage of this CI workflow, the logs are massive, which makes it inconvenient for anyone to read them to identify the cause of a failure. Removing the verbose output will improve that situation

### CODE REVIEW

**Review by facchinm — APPROVED** [review]

**Comment by per1234** [comment]
Hi @facchinm. Just an FYI that I don't have any permissions in this repo so I can't merge this. Maintainers are welcome to merge at their convenience.

**Pull request merged**
